rerun_py/rerun_sdk/rerun/archetypes/depth_image_ext.py

```diff
@@ -115,7 +115,7 @@ def __init__(
         try:
             datatype = ChannelDatatype.from_np_dtype(image.dtype)
         except KeyError:
-            raise ValueError(f"Unsupported dtype {image.dtype} for DepthImage")
+            raise ValueError(f"Unsupported dtype {image.dtype} for DepthImage") from None
```

Comment:
What does `raise X from None` mean? O.o

Reply:
This is a bit of an opinionated lint from Ruff, but I like it because it forces developer to learn about that feature ad think of how to use it.

This is the lint description: https://docs.astral.sh/ruff/rules/raise-without-from-inside-except/

Whether or not to chain exception is a logical/semantic question and only the dev can make the call, so the lint rejects the implicit behaviour.